docs/source/releases/v2.1.rst

```diff
@@ -31,6 +33,46 @@ What's new in Oscar 2.1?
   Django doesn't generate migrations if a project modifies the ``OSCAR_IMAGE_FOLDER``
   to specify a custom directory structure for uploaded images.
 
+- Added ``communication`` app.
+
+- Models ``CommunicationEventType``, ``Email`` and ``Notification`` moved from
+  ``customer`` app to ``communication`` app.
+
+  This is a change that requires database migration.
+
+- ``Dispatcher`` moved from ``customer.utils`` to ``communication.utils``.
+  Now ``Dispatcher`` is responsible for sending notifications as well as emails.
+
+- ``CustomerDispatcher`` util created in ``customer`` app -  to send customer
+  related emails.
+
+- ``AlertsDispatcher`` util created in ``customer.alerts`` module -  to send
+  order product alerts related emails and notifications.
+
+- ``OrderDispatcher`` util created in ``order`` app -  to send order related
+  emails.
+
+- Added ability to chose if sent email should be saved to DB (through
+  ``OSCAR_SAVE_SENT_EMAILS_TO_DB``, ``True`` by default).
+
+- Event codes (for each separate email) now kept in related "Dispatcher".
+
+- Added ability to send emails with attachments.
+
+- All emails' templates (``commtype_*``) moved from ``customer/emails``
+  to ``communication/emails``.
+
+- Templates from ``customer/email/*.html`` and ``customer/notification/*.html``
+  moved to ``communication/email/*.html`` and ``communication/notification/*.html``.
+
+- Models ``CommunicationEventType``, ``Email`` and ``Notification`` added to
+  ``MOVED_MODELS`` to show warning when they will be imported from ``customer`` app.
+  E.g. ``Email = get_model('customer', 'Email')``.
+
+- Added ``absolute_url`` template tag - to get absolute URLs in templates based on
+  provided path and domain. The schema for absolute url can be set through
+  ``OSCAR_URL_SCHEMA`` setting, ``http`` by default.
+
```

Comment:
I've made a couple of comments in the code, but would be grateful if you can add to these notes:

1. All methods that have been removed, and what they have been replaced with.
2. All context processors that have been removed, and what they have been replaced with.

Reply:
Added to this file.